rcljava/src/main/java/org/ros2/rcljava/RCLJava.java

```diff
@@ -255,13 +255,37 @@ public static Node createNode(final String nodeName) {
    * @return A @{link Node} that represents the underlying ROS2 node
    *     structure.
    */
+  public static Node createNode(final String nodeName, final String namespace) {
+    return createNode(nodeName, namespace, new NodeOptions());
+  }
+
+  /**
+   * Create a @{link Node}.
+   *
+   * @param nodeName The name that will identify this node in a ROS2 graph.
+   * @param namespace The namespace of the node.
+   * @param context Context used for creating the node, @link{RCLJava.getDefaultContext()} will be used if `null`.
+   * @return A @{link Node} that represents the underlying ROS2 node
+   *     structure.
+   */
+  // TODO(ivanpauno): Deprecate in favor of @link{RCLJava.createNode(name, namespace, nodeOptionsWithContext)}.
```

Comment:
I think we can add a deprecated annotation now, instead of a TODO.

Reply:
Done in https://github.com/osrf/ros2_java/pull/46/commits/69d508f72b688a27c9b815cd0d04331054035083